test/legacy-html-from-paste.test.js

```diff
@@ -2,6 +2,11 @@
 
 import legacyHtmlFromPaste from '../src/legacy-html-from-paste'
 
+// mock clipboardData.setData function
+window.clipboardData = {}
+window.clipboardData.setData = jest.fn()
+window.clipboardData.setData.mockReturnValue(true)
+
 it('returns the HTML entered by a paste exec command', () => {
   const html = '<p>Some text</p>'
```

Comment:
It'd be useful to have a test to document how this is supposed to work.

Reply:
Update to have tests for both when clipboard is enabled and disabled.